docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Create a Facial Verification System"
+linkTitle: "Verification System"
+type: "docs"
+description: "Configure a verification system module to implement a facial verification security system on your smart machine."
+image: "/tutorials/verification-system/disarmed.png"
+imageAlt: "Bijan disarming the facial verification system."
+images: ["/tutorials/verification-system/disarmed.png"]
+tags: ["mlmodel", "vision", "services", "security", "camera"]
+authors: ["Sierra G."]
+languages: []
+viamresources: ["mlmodel", "vision", "camera"]
+level: "Intermediate"
+date: "28 November 2023"
+cost: "0"
+no_list: true
+# SMEs: Bijan Haney
+---
+
+Follow this tutorial to create your own machine learning (ML) enabled security system, or, "verification system."
+The verification system configuration you will use in this project combines two detectors: The first detector detects whether there are people present, thanks to a person-detection ML model you train yourself.
+You give the second detector photos of known faces and it tells you if a person is recognized.
+
+The verification system has three possible layers of detectors.
+You layer them together so that objects detected in the images your verification's system camera captures point to countdown, alarmed, or disarmed states.
+With our recommended combination of detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
+
+For example:
+
+{{<video webm_src="/tutorials/verification-system/demo.webm" mp4_src="/tutorials/verification-system/demo.mp4" alt="Bijan interacting with the verification system" poster="/tutorials/verification-system/trigger_1.png">}}
```

Comment:
This is a lot better but I think the first sentence really needs to explain the system and I don't think anyone really knows what a verification system is. The rest are fairly minor changes and removing the thing about states which we haven't mentioned yet and the three possible layers. I don't understand that but it's also not explained further - Is that a type?? 

```suggestion
Create a machine learning (ML) enabled security system, or "verification system", that can detect people and verify that the people are meant to be there with facial recognition.
The verification system you will create in this project combines two detectors: 

1. The first detector detects whether there are people present, thanks to a person-detection ML model you train yourself.
2. The second detector verifies that the people present are known.
   You will use an existing model and provide it with a set of photos of known faces.

Layering these two detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.

For example:

{{<video webm_src="/tutorials/verification-system/demo.webm" mp4_src="/tutorials/verification-system/demo.mp4" alt="Bijan interacting with the verification system" poster="/tutorials/verification-system/trigger_1.png">}}

This tutorial uses two layers of detectors to detect people and then faces but if your use case is different, you can adapt the detectors or the layering of the detectors.
```

Reply:
This suggestion looks great, thank you!